sdk/storage/azure-storage-file-share/azure/storage/fileshare/_models.py

```diff
@@ -959,10 +959,6 @@ def _from_generated(cls, generated):
         props.change_time = generated.properties.change_time
         props.etag = generated.properties.etag
         props.permission_key = generated.permission_key
-        props.owner = generated.owner
```

Comment:
Removing `owner`, `group`, `file_mode`, `nfs_file_type`, and `link_count` from generated props as those don't exist in generated models.

Reply:
Do these ever actually get populated for directories? Like in `get_directory_properties`? If not, we should remove them entirely from this model.

Reply:
Yeah they get populated for `get_directory_properties`

Reply:
Ok, weird they aren't in the generated model then. This means they don't get populated for `list_files_and_directories`?